src/couch_replicator.erl

```diff
@@ -76,7 +78,7 @@
     target_monitor = nil,
     source_seq = nil,
     use_checkpoints = true,
-    checkpoint_interval = 5000,
+    checkpoint_interval = ?DEFAULT_CHECKPOINT_INTERVAL,
```

Comment:
You have change `5000` to `30`. That's indeed is timeout reducing, but I fear that in wrong way.